### PR TITLE
Add LDAP how to guide

### PR DESCRIPTION
**Pull request by arturo-seijas**
Add how to guide for LDAP configuration and specifics about SAML configuration

### CODE REVIEW

**Review by merkata — DISMISSED** [review]

**Review by amandahla — DISMISSED** [review]
LGTM

File: docs/how-to-guides/ldap_configuration.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# LDAP configuration
+
+To configure Indico's LDAP integration you'll have to set `ldap_host` with the URL for your LDAP server by running `juju config [charm_name] ldap_host=[value]` and `ldap_password` with its credentials `juju config [charm_name] ldap_password=[value]`.
+
+For more details on the configuration options and their default values see the [configuration reference](https://charmhub.io/indico/configure).
```
Comment (mthaddon) [inline docs/how-to-guides/ldap_configuration.md:5]: The content here is good, but I think it's worth also including information about what LDAP integration provides here, and what the implications of configuring it are. For instance, something like "all LDAP groups from the server you connect to will be imported". Some description of what happens when groups change or users change membership of groups would be useful, and also of how the LDAP instance needs to be configured to work with this.

Comment (arturo-seijas) [reply docs/how-to-guides/ldap_configuration.md]: Fixed

File: docs/how-to-guides/ldap-configuration.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# LDAP configuration
+
+To configure Indico's LDAP integration you'll have to set `ldap_host` with the URL for your LDAP server by running `juju config [charm_name] ldap_host=[value]` and `ldap_password` with its credentials `juju config [charm_name] ldap_password=[value]`.
+
+At the moment, this feature has only been tested against Canonical's LDAP schema. If you are having any issues with your configuration, please, reach out to us.
```
Comment (mthaddon) [inline docs/how-to-guides/ldap-configuration.md:5]: How about "If you are having any issues configuring a different LDAP instance, please, reach out to us."?

Comment (arturo-seijas) [reply docs/how-to-guides/ldap-configuration.md]: Done

**Review by amandahla — DISMISSED** [review]

File: docs/how-to-guides/saml-configuration.md [hunk 3]
```diff
@@ -2,4 +2,6 @@
 
 To configure Indico's SAML integration you'll have to set `saml_target_url` with the URL for your SAML server by running `juju config [charm_name] saml_target_url=[value]`.
 
+At the moment, this feature has only been tested against Ubuntu SSO's SAML implementation. If you are having any issues configuring another SAML provider, please, reach out to us, please, reach out to us.
```
Comment (merkata) [inline docs/how-to-guides/saml-configuration.md:5]: Is this copy paste or do they really need to reach out to us, really need to reach out to us? It sounds intense.

Comment (arturo-seijas) [reply docs/how-to-guides/saml-configuration.md]: Wanted to emphasize :D

**Review by merkata — APPROVED** [review]

**Review by amandahla — APPROVED** [review]

**Review by mthaddon — APPROVED** [review]

**Pull request merged**